python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -35,10 +35,11 @@ def _getName(obj):
     if returnable is None:
         returnable = getattr(getattr(obj, '__class__', None), '__name__', None)
     if returnable is None:
-        returnable = str(obj)
-    return returnable
+        return str(obj)
+    else:
+        return returnable
```

Comment:
This comes about through mangling this function several times after changes weren't wanted, reverting